src/components/case_manager/AllParas.tsx

```diff
@@ -0,0 +1,78 @@
+import { trpc } from "client/lib/trpc";
+import Link from "next/link";
+import React from "react";
+import styles from "../../styles/Home.module.css";
+
+const AllParasPage = () => {
```

Comment:
Please add some documentation about the purpose of this component, and also any props it expects/users.